tests/scripts/components-configuration-crypto.sh

```diff
@@ -2167,6 +2158,8 @@ component_test_ccm_aes_sha256 () {
 
     # Setting a blank config disables everyhing in the library side.
     echo '#define MBEDTLS_CONFIG_H ' >"$CONFIG_H"
+    cp configs/crypto-config-ccm-aes-sha256.h "$CRYPTO_CONFIG_H"
```

Comment:
Nice catch, thanks.

Reply:
Thanks. I've compared the outcomes.csv from before and after #9567 and it was quite clear there was an issue with test_ccm_aes_sha256.